packages/back-end/src/services/organizations.ts

```diff
@@ -346,6 +346,13 @@ function validateConfig(config: ConfigFile, organizationId: string) {
   const errors: string[] = [];
 
   const datasourceIds: string[] = [];
+  if (!config.datasources)
+    errors.push(
+      `Missing object \`datasources\` check your config.yml validity`
+    );
+  if (!config.metrics)
+    errors.push(`Missing object \`metrics\` check your config.yml validity`);
+
```

Comment:
Thinking about this more, importing does a merge with your existing configuration, so I can imagine some scenarios where you would only want to import dimensions for example and not datasources or metrics.  So maybe just remove these checks.

Reply:
I see what you are saying. I am in agreement in that case then. Maybe in the future, this type of check needs to be added after the file is merged if it causes people issues.